recipes/icu/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+cd source
+chmod +x configure install-sh
+
+./configure --prefix="$PREFIX" \
+    --disable-samples \
+    --disable-tests \
```

Comment:
Yeah, looking at the Linux build. It doesn't look like the tests are being run. Could we remove this flag?